.circleci/config.yml

```diff
@@ -31,6 +31,18 @@ commands:
           name: Ensure docker-compose exists, otherwise install it.
           command: ./scripts/docker-compose-check.sh
 
+  docker-compose-up-backend:
```

Comment:
These function the same as before but allow us to prevent repeating the same code.